src/Libraries/DSOffice/DSOfficeImages.resx

```diff
@@ -120,159 +120,158 @@
-      d9fUsGU60Nx3N6UG9wOs+ShYC3zs8dbvG6N8hsPtj9vvrv1x+ccQ4iwT4M+F/JmQfopif0ypwf0Aaz4K
-      1oLTN0+cRjCJB8/v3SIhzjIB7j1rylllPUahxcXGwsh/+zdNRdAdya/dsJjPgr1gYfi2QoOv51l4Xciv
-      3bCO2wsLwx+MDcdMLLwu5Ndu2MLtlxsLwvd6clhwXaoFLSw3lsPBURwJf3FioXWp/DlwEG0sLx7CKIuE
-      r9dZaF0qfydYnAlgS7goWHdgIfgef+Ld9tvXbBnpsmEW5jTL0Ebt+wA7O4hCm/2HNR8FawHySLAAXTaU
-      h9iKeaNCTQvrMQot9h/WfBSsBf+9f9Ougjae3r46QYitmDcq858yNQxYj1FocbGxMPId0C424MYp35jT
-      zmiDheEnXw3+Gz3Jvv+0ivtBOinLsUDyo8dLJhbiLPJrNgxwW9FgocSzgC7fiP3IMucIbityLJh2S9rw
-      n4suJsGvkf/0bOB2ImLhtO8EYNZJiOE7i7myYUu4KFgLcs/Nx1d+QlAJP5x59M/DO/aRBTx2Geo1Xmse
-      RvoJyu87TqnB/QBrPgrWguiLkwB8D8cPVXaskPLQ/RTND3P8jdrGIvH/RUfifZnQYv9hzUfBWsC8F3/9
-      4bv8jDjg+zm+qln788Xg7pN//77vZ8hpJICXPN9v8l3X/Bkz9Ilm986FFpcTy8hDG/cHrUnw2vbvif7Z
-      tGMSDK2IdsNCOmDyMCf5m6J7/A+86Z8PEftekzALFpZPhr85e8i5/DsaesR8moR5Y4FrEuaNBa5JmDcW
-      uCZh3ljgmoR5Y4FrEuaNBa5JmDcWeJwEnZx9biz0ZhIU/rzwScBHIYQQQgghhBBCCCGEEEIIIcQUrKx8
-      AuDviEpOlLH+AAAAAElFTkSuQmCC
-    </value>
+        iVBORw0KGgoAAAANSUhEUgAAAGAAAABgCAYAAADimHc4AAAABGdBTUEAALGPC/xhBQAAABl0RVh0U29m
+        dHdhcmUAQWRvYmUgSW1hZ2VSZWFkeXHJZTwAAASGSURBVHhe7ZpPi1RHFMXnI8xH8CPkI/gFBJdZCm6y
+        EpfBlRvBrHQVBBcGEmYVnLhxIYriRlzIhIiJSmJLEtQByajRxD8j7b3V5z1qbp2u6Z5+2G+6zw8ONF3n
+        vnc5Bf3qVfWKEEIIIYQQQgghhBBCCCGEEEIIIYQQQkzC11e+Gk4jlCXYeE0oS7DxmlCWYONRsPYf1nxN
+        KEuw8ZpQlmDjNaEswcajYO0/rPmaUJZg4zWhLMHGa0JZgo1Hwdp/WPM1oSzBxmtCWYKN14SyBBuPgrX/
+        sOZrQlmCjdeEsgQbrwllCTYeBWv/Yc3XhLIEG68JZQk2XhPKEmw8ClYhhBBCCNFPbMl2cjfBWsC8UbAW
+        MG8UrAXMGwVr/2HNR8FawLxRsBYwbxSsBcwbBWv/Yc1HwVrAvFHuGw6HX5hOmtZN113P32wOXE9e/fXb
+        L8/u3Pj+53PnWC0j+phgXV4s5FXTcdPANCnu9ZpVXEbsBQvwMMLcK157GJcT02DBnUkRltwynTedCPLv
+        fIxxBpcVk2CBXRjl1vLatGb60nRoF7nHvV6TcwGXFzU8qFFeLY9MR00s7Jq8xmtzNAk1LCB/cOZcNbFw
+        p5FfI+c4brdYsCVcFKwFPvbt7W/Ofvj4/n+ENHz19uXm5YcX1xrZVyzcQ7lnnMx319SwZTrQ3Hc3pQb3
+        A6z5KFgLfOzx1u8bo3yGw+2P2++u/XH5xxDiLBPgz4X8mZB+imJ/TKnB/QBrPgrWgtM3T5xGMIkHz+/d
+        IiHOMgHuPWvKWWU9RqHFxcbCyH/7N01F0B3Jr92wmM+CvWBh+LZCg6/nWXhdyK/dsI7bCwvDH4wNx0ws
+        vC7k127Ywu2XGwvC93pyWHBdqgUtLDeWw8FRHAl/cWKhdan8OXAQbSwvHsIoi4Sv11loXSp/J1icCWBL
+        uChYd2Ah+B5/4t3229dsGemyYRbmNMvQRu37ADs7iEKb/Yc1HwVrAfJIsABdNpSH2Ip5o0JNC+sxCi32
+        H9Z8FKwF/71/066CNp7evjpBiK2YNyrznzI1DFiPUWhxsbEw8h3QLjbgxinfmNPOaIOF4SdfDf4bPcm+
+        /7SK+0E6KcuxQPKjx0smFuIs8ms2DHBb0WChxLOALt+I/cgy5whuK3IsmHZL2vCfiy4mwa+R//Rs4HYi
+        YuG07wRg1kmI4TuLubJhS7goWAtyz83HV35CUAk/nHn0z8M79pEFPHYZ6jVeax5G+gnK7ztOqcH9AGs+
+        CtaC6IuTAHwPxw9VdqyQ8tD9FM0Pc/yN2sYi8f9FR+J9mdBi/2HNR8FawLwXf/3hu/yMOOD7Ob6qWfvz
+        xeDuk3//vu9nyGkkgJc832/yXdf8GTP0iWb3zoUWlxPLyEMb9wetSfDa9u+J/tm0YxIMrYh2w0I6YPIw
+        J/mbonv8D7zpnw8R+16TMAsWlk+Gvzl7yLn8Oxp6xHyahHljgWsS5o0FrkmYNxa4JmHeWOCahHljgWsS
+        5o0FrkmYNxZ4nASdnH1uLPRmEhT+vPBJwEchhBBCCCGEEEIIIYQQQgghxBSsrHwC4O+ISk6Usf4AAAAA
+        SUVORK5CYII=
```

Comment:
The content for this one looks different too. Is that expected?

Reply:
Yea not sure whats going on here, haven't touched this either

Reply:
I checked and in Dynamo the icons of the node look ok, so this is harmless.